<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vehicleWidget</class>
 <widget class="QWidget" name="vehicleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>431</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="nameTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="arrivalTextEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="arrivalLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arrival</string>
    </property>
   </widget>
   <widget class="QLabel" name="departureLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Departure</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="capacityLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="capacityTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chargePowerLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ChargePower</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="capacityTextEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="capacityTextEdit_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SOCLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SOC</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="capacityTextEdit_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="desiredChargeLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DesiredCharge</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
